lib/utils.js

```diff
@@ -1,5 +1,26 @@
 let { list } = require('postcss')
 
+class Cache {
+  constructor () {
```

Comment:
What does this wrap give to us? For me, it will just slow down the calculation.

I think for the few use cases, isolation from the wrapper class is not necessary.